includes/functions.php

```diff
@@ -101,8 +101,9 @@ function getHostOS($device)
 
     $sysDescr    = snmp_get($device, "SNMPv2-MIB::sysDescr.0", "-Ovq");
     $sysObjectId = snmp_get($device, "SNMPv2-MIB::sysObjectID.0", "-Ovqn");
+    $sysMfgName  = snmp_get($device, "ENTITY-MIB::entPhysicalMfgName.1", "-Ovqn");
```

Comment:
Here yes, we also use it in siklu discovery but in other vendors no it can be quit varied, Cisco seems to be [0-9]!